tests/run-make/core-no-fp-fmt-parse/rmake.rs

```diff
@@ -0,0 +1,17 @@
+// This test checks that the core library of Rust can be compiled without enabling
+// support for formatting and parsing floating-point numbers.
+
+extern crate run_make_support;
+
+use run_make_support::rustc;
+use std::path::PathBuf;
+
+fn main() {
+    rustc()
+        .edition("2021")
+        .arg("-Dwarnings")
+        .crate_type("rlib")
+        .arg_path("../../../library/core/src/lib.rs")
```

Comment:
This is the input file right? In that case, you don't need to introduce `arg_path` yet (You can just do `input("../../../library/core/src/lib.rs")`).

Reply:
It... seems that `arg_path` and `input` functions are actually the exact same? 

```rust
    pub fn input<P: AsRef<Path>>(&mut self, path: P) -> &mut Self {
        self.cmd.arg(path.as_ref());
        self
    }
```

and

```rust
    pub fn arg_path<P>(&mut self, path: P) -> &mut Self
    where
        P: AsRef<Path>,
    {
        self.cmd.arg(path.as_ref());
        self
    }
```

It might be worth keeping only one.

Reply:
Yeah, use `input` for now because it's semantically meaningful, when/if you need `arg_path` for another test then you could add it if necessary / no other meaningful helper methods exist.

Reply:
Very good, I have removed `arg_path` and changed to `input` in the test.